src/main/resources/assets/tectech/lang/en_US.lang

```diff
@@ -775,9 +775,9 @@ gt.blockmachines.debug.tt.data.name=Debug Data Hatch
 gt.blockmachines.debug.tt.data.desc.0=Quantum Data Output
 gt.blockmachines.debug.tt.data.desc.1=High speed fibre optics connector.
 gt.blockmachines.debug.tt.data.desc.2=Must be painted to work
-gt.blockmachines.debug.tt.maintenance.name=Debug Maintenance Hatch
-gt.blockmachines.debug.tt.maintenance.desc.0=For maintaining Multiblocks
-gt.blockmachines.debug.tt.maintenance.desc.1=Does fix everything but itself.
+gt.blockmachines.debug.tt.maintenance.name=Auto-Taping Maintenance Hatch
+gt.blockmachines.debug.tt.maintenance.desc.0=For automatically maintaining Multiblocks
+gt.blockmachines.debug.tt.maintenance.desc.1=Fixes everything including itself.
```

Comment:
Undo. This is a joke, I know they are hard but don't change this!!

Reply:
Did you just want me to change line 780 back? I'm not especially familiar with github reviews. Or were you joking about renaming the block too?